static/i18n/de/app.json

```diff
@@ -890,7 +890,9 @@
         "title": "Account",
         "label": "Account to credit",
         "parentAccount": "1. Select {{currencyName}} account",
-        "token": "2. Select a token"
+        "token": "2. Select a token",
+        "verifyTokenType": "Please verify that <bold>{{token}}</bold> is an <bold>{{tokenType}}</bold> token. Any other crypto asset sent to an {{currency}} account may be lost.",
```

Comment:
I'm not sure if the <bold> tag is handled specifically by i18n :thinking:  if so keep it or use <0>...</0> instead with inside the translated component 
```
<Trans i18nKey="..." >
<b/>
</Trans>
```

Reply:
i was using this method https://react.i18next.com/latest/trans-component#alternative-usage-v-11-6-0, but i switched to the lld project method